[BUG] 403 Forbidden when using V-Host

**Swagger-Promote version**
> Please make sure to use the latest Swagger-Promote version.

**API-Manager and Service-Pack Version**
7.7.20200130

**Expected behavior**
vhost should be set

**Actual behavior**
swagger promote CLI is failing and rolling back the changes. 

**Log-Output in Debug**
```
10679  ERROR          RestAPICall| Response: 'HTTP/1.1 403 Forbidden'
11095  INFO        RollbackAction| Rollback FE-API: 'Axway Imagine - Weather API' (ID: 'd3c6beb2-1e57-461c-92e2-18813302c5e7')
11539  INFO               Proxies| Sending request to find existing APIs: https://35.171.233.150:8075/api/portal/v1.3/apirepo?field=name&op=like&value=Axway+Imagine+-+Weather+API+HTTP&field=createdOn&op=gt&value=1588650992895
11770  INFO               Proxies| No existing API found based on filter: [ filter=[field=name, op=like, value=Axway Imagine - Weather API HTTP, field=createdOn, op=gt, value=1588650992895]]
11770  INFO       RollbackHandler| Rolled back: '[Frontend-API: true, Backend-API: true]'
11771  ERROR                  App| Cannot update API-Proxy.
com.axway.apim.lib.errorHandling.AppException: Cannot update API-Proxy.
	at com.axway.apim.apiimport.actions.tasks.UpdateAPIProxy.execute(UpdateAPIProxy.java:56)
	at com.axway.apim.apiimport.actions.tasks.props.VhostPropertyHandler.handleVHost(VhostPropertyHandler.java:49)
	at com.axway.apim.apiimport.actions.tasks.props.VhostPropertyHandler.handleVHost(VhostPropertyHandler.java:37)
	at com.axway.apim.apiimport.actions.CreateNewAPI.execute(CreateNewAPI.java:114)
	at com.axway.apim.apiimport.APIImportManager.applyChanges(APIImportManager.java:49)
	at com.axway.apim.App.run(App.java:247)
	at com.axway.apim.App.main(App.java:53)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.codehaus.mojo.exec.ExecJavaMojo$1.run(ExecJavaMojo.java:282)
	at java.lang.Thread.run(Thread.java:748)
Caused by: com.axway.apim.lib.errorHandling.AppException: Unable to parse HTTP-Response
	at com.axway.apim.lib.utils.rest.RestAPICall.parseResponse(RestAPICall.java:73)
	at com.axway.apim.lib.utils.rest.PUTRequest.execute(PUTRequest.java:28)
	at com.axway.apim.apiimport.actions.tasks.UpdateAPIProxy.execute(UpdateAPIProxy.java:54)
	... 12 more
Caused by: com.axway.apim.lib.errorHandling.AppException: Unable to parse response
	at com.axway.apim.apiimport.actions.tasks.UpdateAPIProxy.parseResponse(UpdateAPIProxy.java:81)
	at com.axway.apim.lib.utils.rest.RestAPICall.parseResponse(RestAPICall.java:65)
	... 14 more
Caused by: java.lang.ClassCastException: java.lang.String cannot be cast to org.apache.http.client.methods.HttpEntityEnclosingRequestBase
	at com.axway.apim.apiimport.actions.tasks.UpdateAPIProxy.parseResponse(UpdateAPIProxy.java:77)
	... 15 more
```

**Test case sample**
> Please describe the steps you have done to replicate the issue
Add a vhost element in config.json to simulate the issue

--- a/modules/swagger-promote-core/src/main/java/com/axway/apim/apiimport/actions/CreateNewAPI.java
+++ b/modules/swagger-promote-core/src/main/java/com/axway/apim/apiimport/actions/CreateNewAPI.java
@@ -111,7 +111,8 @@ public class CreateNewAPI {
 			new ManageClientApps(changes.getDesiredAPI(), createdAPI, changes.getActualAPI()).execute(reCreation);
 			
 			// V-Host must be managed almost at the end, as the status must be set already to "published"
-			vHostHandler.handleVHost(changes.getDesiredAPI(), createdAPI);
+			//vHostHandler.handleVHost(changes.getDesiredAPI(), createdAPI);
+			// Vhost should be setup during the api publish
 		} catch (Exception e) {
 			throw e;
 		} finally {

--- a/modules/swagger-promote-core/src/main/java/com/axway/apim/apiimport/actions/CreateNewAPI.java
+++ b/modules/swagger-promote-core/src/main/java/com/axway/apim/apiimport/actions/CreateNewAPI.java
@@ -110,9 +110,7 @@ public class CreateNewAPI {
 			// Handle subscription to applications
 			new ManageClientApps(changes.getDesiredAPI(), createdAPI, changes.getActualAPI()).execute(reCreation);
 			
-			// V-Host must be managed almost at the end, as the status must be set already to "published"
-			//vHostHandler.handleVHost(changes.getDesiredAPI(), createdAPI);
-			// Vhost should be setup during the api publish
+			// The V-Host is managed in UpdateAPIStatus
 		} catch (Exception e) {
 			throw e;
 		} finally {

--- a/modules/swagger-promote-core/src/test/resources/com/axway/apim/test/files/vhost/1_vhost-config.json
+++ b/modules/swagger-promote-core/src/test/resources/com/axway/apim/test/files/vhost/1_vhost-config.json
@@ -1,9 +1,19 @@
-	
 {
-	"name": "${apiName}",
-	"path": "${apiPath}",
-	"state": "${status}",
-	"vhost": "${vhost}",
-	"version": "1.0.0",
-	"organization": "API Development ${orgNumber}"
+  "name": "${apiName}",
+  "path": "${apiPath}",
+  "state": "${status}",
+  "vhost": "${vhost}",
+  "version": "1.0.0",
+  "organization": "API Development ${orgNumber}",
+  "authenticationProfiles": [
+    {
+      "name": "_default",
+      "isDefault": true,
+      "parameters": {
+        "username": "daskdjaskldjaskljdklasjdl",
+        "password": ""
+      },
+      "type": "http_basic"
+    }
+  ]
 }

